components/Pagination/Pagination.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import { number, string } from 'prop-types';
+import classNames from 'classnames';
+import LeftAngleIcon from 'static/images/icons/FontAwesome/angle-left-solid.svg';
+import RightAngleIcon from 'static/images/icons/FontAwesome/angle-right-solid.svg';
+import PaginationItem from './PaginationItem/PaginationItem';
+import styles from './Pagination.css';
+
+Pagination.propTypes = {
+  className: string,
+  currentPage: number.isRequired,
+  totalPages: number.isRequired,
+};
+
+Pagination.defaultProps = {
+  className: undefined,
+};
+
+function Pagination({ className, currentPage, totalPages }) {
+  const PaginationItems = () => {
+    const shouldTruncateStart = currentPage - 1 > 5;
+    const shouldTruncateEnd = totalPages - currentPage > 5;
+
+    const paginationStart = shouldTruncateStart ? currentPage - 4 : 1;
+    const paginationEnd = shouldTruncateEnd ? currentPage + 4 : totalPages;
+
+    const paginationLength = paginationEnd - paginationStart + 1;
+
+    const PaginationItemArray = [...Array(paginationLength)].map((_, index) => {
+      const page = index + paginationStart;
+      const isCurrent = page === currentPage;
+
+      return <PaginationItem key={page} value={page} isCurrent={isCurrent} />;
+    });
+
+    return (
+      <>
```

Comment:
Can you make this an `ol` element and make the `PaginationItem` component a `li` element?

You'll have to do some style overrides, but it'll be really semantic HTML 💃

Reply:
Thanks for your input. Yeah I saw the tests, it reduced the complexity but this time another test failed because I forgot to update the storybook snapshot. 
Also I have added some tests for `PaginationItem`.